components/suggest/src/db.rs

```diff
@@ -306,6 +306,12 @@ impl<'a> SuggestDao<'a> {
                         let cooked_url = cook_raw_suggestion_url(&raw_url);
                         let raw_click_url = row.get::<_, String>("click_url")?;
                         let cooked_click_url = cook_raw_suggestion_url(&raw_click_url);
+                        let icon_data = row.get::<_, Option<_>>("icon_data")?;
+                        let icon_mimetype = row.get::<_, Option<_>>("icon_mimetype")?;
+                        let icon = icon_data.map(|data| SuggestionIcon {
+                            data,
+                            mimetype: icon_mimetype.unwrap_or_else(|| "".into()),
```

Comment:
```suggestion
                            mimetype: icon_mimetype.unwrap_or_default(),
```

I _think_ this should work; please ignore if it doesn't!

Reply:
Thanks! This works!